server/manager/src/main/java/org/apache/accumulo/manager/ManagerClientServiceHandler.java

```diff
@@ -96,15 +96,15 @@
 
 import com.google.protobuf.InvalidProtocolBufferException;
 
-public class ManagerClientServiceHandler extends FateServiceHandler
-    implements ManagerClientService.Iface {
+public class ManagerClientServiceHandler implements ManagerClientService.Iface {
 
   private static final Logger log = Manager.log;
   private static final Logger drainLog =
       LoggerFactory.getLogger("org.apache.accumulo.manager.ManagerDrainImpl");
+  protected final Manager manager;
```

Comment:
I don't think this needs to be protected anymore. Can probably be private now, or package-private. Same with the one in FateServiceHandler.java.

Reply:
Addressed in e243d49 and c0814db